src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -123,7 +123,7 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
     }
 
     private createChart() {
-        const labels = [];
+        const labels: any[] = [];
```

Comment:
This looks like it can better be string[].
```suggestion
        const labels: string[] = [];
```